tests/python/frontend/onnx/test_forward.py

```diff
@@ -3296,6 +3299,15 @@ def verify_lppool(x_shape, kernel_shape, p, strides, pads, out_shape, auto_pad="
         out_shape=[1, 1, 16, 16, 16],
         auto_pad="SAME_UPPER",
     )
+    # Pool2D with empty p
+    verify_lppool(
+        x_shape=[1, 1, 32, 32],
+        kernel_shape=[3, 3],
+        p=None,
+        strides=[1, 1],
+        pads=[1, 1, 1, 1],
+        out_shape=[1, 1, 31, 31],
```

Comment:
Yeah, with unity strides, I'm pretty sure the output shape here should be [1, 1, 32, 32]